webhook_verification.go

```diff
@@ -0,0 +1,68 @@
+package ngroklistener
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/caddyconfig/caddyfile"
+	"golang.ngrok.com/ngrok/config"
+)
+
+type webhookVerification struct {
+	WebhookVerificationOption config.HTTPEndpointOption
```

Comment:
Can you un-export this? There's no obvious need for this to be exported, so I prefer to reduce surface exposure. Am I missing a particular need for this?

Reply:
That one is because of the flow 
```
if t.WebhookVerification != nil {
		err := t.WebhookVerification.Provision(ctx)
		if err != nil {
			return fmt.Errorf("provisioning webhook_verification: %v", err)
		}
		t.opts = append(t.opts, t.WebhookVerification.WebhookVerificationOption)
	}
```
Matching how OAuth and OIDC were handled.

Reply:
They're still accessible when unexported. I might've missed the similar fields being exported in those other options 😅

Reply:
Done! Fixed OIDC and OAuth as well :)